src/System.Windows.Forms/src/System/Windows/Forms/Control.ActiveXImpl.cs

```diff
@@ -1307,22 +1304,22 @@ internal void OnAmbientPropertyChange(DispatchID dispID)
                     }
 
                     // Special properties that we care about
-                    object obj = new object();
+                    object? obj = new object();
```

Comment:
```suggestion
                    object? obj = null;
```

No reason to allocate something that is never used.